testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -138,7 +140,6 @@ def _wait_for_containers(self, expected_containers):
             )
             running_countainers_count = len(out.split())
             if running_countainers_count == expected_containers:
-                time.sleep(60)
```

Comment:
This I don't feel confident with merging, as it is one of this sleeps that removing them made tests flaky. I suggest you submit this in a separate PR and run the full test suite like 10 times.

Reply:
I reverted this, perhaps we can use the new liveliness endpoints to wait for the back-end to start?